recipes/default.rb

```diff
@@ -22,6 +22,16 @@
   not_if { node['consul']['create_service_user'] == false }
 end
 
+# enable client streaming based on node's superfarm
+# FIXME: remove after streaming is on for all superfarms
+if node['consul']['config']['use_streaming_backend'] == false
+  if [
+    'AssetDelivery',
```

Comment:
do we have a way to write a unit test for this ? I'm not familiar with this cookbook, so I don't know how hard it is to create a spec for this.

Reply:
I'll test it in alpha

Reply:
ok, I'm not familiar with ruby, but this quick test seems to validate the syntax:
```
irb(main):001:0> superfarm = "test"
=> "test"
irb(main):002:0> ['test'].include?superfarm
=> true
```